Firestore/core/src/firebase/firestore/util/executor_libdispatch.mm

```diff
@@ -71,6 +72,11 @@ void DispatchSync(const dispatch_queue_t queue, std::function<void()> work) {
   });
 }
 
+}  // namespace internal
+
+using internal::DispatchAsync;
+using internal::DispatchSync;
```

Comment:
Consider moving these 'using' decl's into the anon namespace, 2 lines down. (http://go/totw/119#scope-of-the-alias)

Reply:
Done.